src/Core/tests/DeviceTests/Stubs/EntryStub.cs

```diff
@@ -11,5 +11,17 @@ public partial class EntryStub : StubBase, IEntry
 		public bool IsPassword { get; set; }
 		
 		public bool IsTextPredictionEnabled { get; set; }
+
+		public Keyboard Keyboard { get; set; }
+
+		public bool IsSpellCheckEnabled { get; set; }
+
+		public int MaxLength { get; set; }
+
+		public string Placeholder { get; set; }
+
+		public Color PlaceholderColor { get; set; }
```

Comment:
Also remove color from this PR. The first one to get merged will win.

Reply:
It is currently coming from ITextInput, and IsReadOnly is also a part of ITextInput. Still not sure what to do with them :(

Reply:
I will remove the usage of ITextInput, since we don't have a clear plan on it, so I will add this property without the Interface, and later on, we can refactor this code and add ITextInput or some other related interface.

Reply:
@jsuarezruiz @mattleibow can you please, help me with the instructions on what to do with ITextInput, to use it or not... I would like to finish this PR 😄